intelmq/bots/collectors/alienvault_otx/OTXv2.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import httplib
+import urlparse
+import urllib
+import urllib2
+import simplejson as json
```

Comment:
One last question: why did you chose simplejson over json? To cite the [release notes of Python 2.7](https://docs.python.org/3.4/whatsnew/2.7.html?highlight=simplejson#new-and-improved-modules):

> Updated module: The json module was upgraded to version 2.0.9 of the simplejson package, which includes a C extension that makes encoding and decoding faster. (Contributed by Bob Ippolito; issue 4136.)

Also see [SO: 
What are the differences between json and simplejson Python modules?
](https://stackoverflow.com/questions/712791/what-are-the-differences-between-json-and-simplejson-python-modules), links a unicode bug in simplejson and states:
>  json is simplejson, added to the stdlib.